Users/Install.rst

```diff
@@ -280,6 +280,13 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Integrating Shell Autocompletion
+--------------------------------
+
+You can also use coala for tab completion in bash or zsh.
+This will autocomplete coala arguments and bear names.
+You can set them up `here <http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html>`_.
```

Comment:
Tutorials/Shell_Autocompletion is an internal documentation that can be cross-referenced using sphinx role :doc: as described here: http://www.sphinx-doc.org/en/1.5.1/markup/inline.html#role-doc

Can you try doing that, instead of using an external link? 

( http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html is considered an external link )

Perhaps also rephrase this to:
```
You can set them up by following this tutorial. (tutorial as hyperlink)
```

After that, please use squash all of your commits into one.
Instructions for squashing commits are here:
http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

Thanks.